test_nihalo.py

```diff
@@ -0,0 +1,108 @@
+import unittest
+
+from nihalo import *
+import numpy as np
+
+# Metodo Runge Kutta 4 testiramo tako, da rešimo nekaj diferencialnih enačb,
```

Comment:
Dober način testiranja. 

Dodatno se pri metodah za integracijo in NDE splača testirati, če je implementacija metode dovolj visokega reda. To lahko preprosto storimo tako, da podvojimo število korakov in preverimo da se napaka zmanjša za približno 2 na red metode. Pri metodah za integracijo in NDE se napake v implementaciji namreč pogosto odražajo v znižanju reda metode.